website/content/api-docs/kv.mdx

```diff
@@ -29,6 +29,12 @@ This endpoint returns the specified key. If no key exists at the given path, a
 For multi-key reads (up to a limit of 64 KV operations) please consider using
 [transactions](/api/txn) instead.
 
+If the [`recurse`](#recurse) or [`keys`](#keys) query parameters are `true`,
```

Comment:
@trujillo-adam : this is the only endpoint that I didn't think could reference the partial (because the endpoint only returns an array of keys if certain query parameters are used, not in all cases).

Reply:
EDIT: one more exception was added (Execute Prepared Query)